templates/recovery.conf.epp

```diff
@@ -0,0 +1,64 @@
+<%- |
+  Optional[String]    $restore_command,
+  Optional[String[1]] $archive_cleanup_command,
+  Optional[String[1]] $recovery_end_command,
+  Optional[String[1]] $recovery_target_name,
+  Optional[String[1]] $recovery_target_time,
+  Optional[String[1]] $recovery_target_xid,
+  Optional[Boolean]   $recovery_target_inclusive,
+  Optional[String[1]] $recovery_target,
+  Optional[String[1]] $recovery_target_timeline,
+  Optional[Boolean]   $pause_at_recovery_target,
```

Comment:
I always find `Optional[Boolean]` interesting because it's usually equal to `Boolean $var = false`. Not always, since you can have a tri-state where undefined is special cased, but not here. So I'd suggest to simplify it here.